modules/ww-ena/README.md

```diff
@@ -0,0 +1,388 @@
+  }
+
+  # Run quality control on downloaded files
+  scatter (fastq_file in download_files.downloaded_files) {
+    call fastqc_tasks.fastqc {
+      input:
+        fastq = fastq_file
+    }
+  }
+
+  output {
+    Array[File] qc_reports = fastqc.html_report
+  }
+}
+```
+
+## Testing the Module
+
+The module includes a test workflow (`testrun.wdl`) that can be run independently:
+
+```bash
+# Using miniWDL
+miniwdl run testrun.wdl
+
+# Using Sprocket
+sprocket run testrun.wdl --entrypoint ena_example
+
+# Using Cromwell
+java -jar cromwell.jar run testrun.wdl
+```
+
+### Validation
+
+The test workflow includes a validation task that checks:
+- Files were successfully downloaded
+- Output files are non-empty
+- Download logs and summaries are generated
+
+## Docker Container
+
+This module uses the `getwilds/ena-tools:2.1.1` container image from the [WILDS Docker Library](https://github.com/getwilds/wilds-docker-library), which includes:
+- ena-file-downloader tool (version 2.1.1)
+- Java runtime environment
+- Bash shell
+- Standard Unix utilities
+
+## ENA Data Access
+
+### Finding Accession Numbers
+
+ENA accession numbers can be found through:
+- **ENA Browser**: https://www.ebi.ac.uk/ena/browser/
+- **ENA Search**: https://www.ebi.ac.uk/ena/browser/advanced-search
+
+Common accession formats:
+- Study: `PRJEB*`, `PRJNA*`, `ERP*`, `SRP*`
+- Sample: `SAMEA*`, `SAMN*`, `ERS*`, `SRS*`
+- Run: `ERR*`, `SRR*`, `DRR*`
+
+### File Formats
+
+The module supports multiple data formats:
+- **READS_FASTQ**: Most common format for sequencing reads
+- **READS_SUBMITTED**: Original submitted format (may be FASTQ, FASTA, or other)
+- **READS_BAM**: Aligned reads in BAM format
+- **ANALYSIS_SUBMITTED**: Analysis files as submitted
+- **ANALYSIS_GENERATED**: ENA-generated analysis files
+
+## Performance Considerations
+
+### Transfer Protocols
+
+- **FTP (default)**: Reliable, works everywhere, moderate speed
+- **ASPERA**: Much faster for large files, requires Aspera Connect/CLI installation
+
+### Resource Requirements
+
+Default settings work well for most use cases:
+- **CPU**: 2 cores (downloads are not CPU-intensive)
+- **Memory**: 8 GB (sufficient for download management)
+- **Disk**: Varies by data size; ensure adequate storage for your downloads
+
+### Runtime Estimates
+
+Download times depend on:
+- File sizes (FASTQ files typically 1-10 GB per sample)
+- Network speed
+- Transfer protocol (Aspera is significantly faster)
+- ENA server load
+
+Typical download speeds:
+- **FTP**: 10-50 MB/s
+- **ASPERA**: 100-500 MB/s (with good network connection)
+
+## Citation
+
+If you use this module in your research, please cite:
+
+**ENA Database:**
+> Leinonen R, Sugawara H, Shumway M; International Nucleotide Sequence Database Collaboration.
+> The sequence read archive. Nucleic Acids Res. 2011 Jan;39(Database issue):D19-21.
+> DOI: 10.1093/nar/gkq1019
+
+**ena-file-downloader tool:**
+> https://github.com/enasequence/ena-ftp-downloader
```

Comment:
```suggestion
```

I don't think this is the right citation, seems to be focused on SRA? Also, I wouldn't expect a paper to describe or cite a downloader they used. I did find this page about ENA and citations but that seems more focused on depositing data: https://ena-browser-docs.readthedocs.io/en/latest/about/citing-ena.html